comic_dl/sites/manganelo.py

```diff
@@ -25,8 +25,16 @@ def __init__(self, manga_url, download_directory, chapter_range, **kwargs):
                              download_directory=download_directory, chapter_range=chapter_range, conversion=conversion,
                              keep_files=keep_files)
 
-    def single_chapter(self, manga_url, download_directory, conversion, keep_files):
-        source, cookies = globalFunctions.GlobalFunctions().page_downloader(manga_url=manga_url)
+    def single_chapter(self, manga_url, download_directory, conversion, keep_files, referer=None):
+        if referer is not None:
```

Comment:
This can be simplified:
```
appended_headers = None
if referer:
    appended_headers = {
                'referer': referer
            }
source, cookies = globalFunctions.GlobalFunctions().page_downloader(manga_url=manga_url,append_headers=appended_headers)

```

Reply:
done